charts/rd-commondata-api/values.yaml

```diff
@@ -6,8 +6,8 @@ java:
   image: 'hmctspublic.azurecr.io/rd/commondata-api:latest'
 
   environment:
-    POSTGRES_HOST: rd-commondata-api-postgres-db-v11-{{ .Values.global.environment }}.postgres.database.azure.com
-    POSTGRES_USERNAME: dbcommondata@rd-commondata-api-postgres-db-v11-{{ .Values.global.environment }}
+    POSTGRES_HOST: rd-commondata-api-postgres-db-v14-{{ .Values.global.environment }}.postgres.database.azure.com
+    POSTGRES_USERNAME: dbcommondata@rd-commondata-api-postgres-db-v14-{{ .Values.global.environment }}
```

Comment:
we don't need to change these at the moment

Reply:
reverted